features/steps/test_post_endpoint.rb

```diff
@@ -0,0 +1,42 @@
+class Spinach::Features::TestPostEndpoint < Spinach::FeatureSteps
+  step 'I create an Item' do
+    HTTParty.post('http://localhost:3000/api/v1/items', {body: {name: 'Foo', quality: 4, sellIn: 4}})
```

Comment:
Comment, No Change Required: You repeat `http://localhost:3000` a number of times in this file. Why not create a helper function:

```
def base_url()
  http://localhost:3000
end
```

Then you can do something like:

```
url = base_url() + '/api/v1/items'
HTTParty.post(url, {...})
```